index.html

```diff
@@ -0,0 +1,79 @@
+
+  <title>Story</title>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;1,400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+</head>
+<body>
+  <main>
+    <div class="map-container">
+      <div class="map" id="map"></div>
+    </div>
+    <section class="slide-section">
+      <article class="slide" id="1-PA-slide">
+        <h2>Food Insecurity in Philadelphia, Pennsylvania</h2>
+        <p>Food insecurity is defined by the US Department of Agriculture as “the lack of access, at times, to enough food for an active, healthy life”. This storymap examines food insecurity in <span style="color: blue">Philadelphia</span>. The food insecurity rate in Philadelphia was 15.2% in 2022, the third highest level across the entire state of Pennsylvania. Having access to food is a basic human right, yet many in the city are still struggling to access sufficient, affordable, and healthful food. Scroll down to learn more about how this issue varies across the different neighborhoods of Philadelphia, and what you can do to help.</p>
+        <p><i>Data source: Feeding America</i></p>
+      </article>
+
+      <article class="slide" id="2-Poverty-slide">
+        <h2>Factors affecting food insecurity: Poverty </h2>
+        <p>What exactly are the factors that contribute to food insecurity? This map shows the percentage of households whose income was below the poverty level in 2022. Poverty is one of the main drivers of food insecurity, as limited income affects one's ability to purchase sufficient food, or nutritious food which tends to be more expensive. Low-income households may opt for cheaper processed foods over fresh fruits or vegetables, which can lead to poor nutrition. The north and west regions of Philadelphia experience comparatively greater levels of poverty.</p>
+        <p><i>Data source: American Community Survey 2022 5-year estimates</i></p>
+      </article>
+      
+      <article class="slide" id="3-CB-slide">
+        <h2>Factors affecting food insecurity: Rent Cost-Burden</h2>
+        <p>Another huge contributor to food insecurity is housing cost-burden. Households are cost-burdened if housing costs take up at least 30% of their entire household income. This reduces  the disposable income remaining to spend on food. With such a large percentage of their income going into paying for rent, these households are sometimes forced to sacrifice food on the table for a roof over their heads. Cost-burdened households generally correlate with poverty levels, with higher percentages in north and west Philadelphia.</p>
+        <p><i>Data source: American Community Survey 2022 5-year estimates</i></p>
+      </article>
```

Comment:
**suggestion:** It would be cool if there were a legend somewhere in each of these slides explaining the colors displayed on the map.